src/components/Card/Card.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import propTypes from "prop-types";
+import classNames from "classnames";
+import styles from "./Card.module.scss";
+import { Text } from "../Text/Text";
+
+export function Card({
+  disabled = false,
+  primaryText = "",
```

Comment:
I'm just thinking about the design of the component - does it make sense to not pass `primaryText`? Maybe it would be worth it for some props to be required?

Reply:
Totally agree